src/it/basic/build.xml

```diff
@@ -33,10 +35,20 @@
 
         <wlp:install-feature ref="testServer" name="mongodb-2.0" whenFileExists="ignore" acceptLicense="true" />
         <!-- install the same feature twice - should not cause an error -->
-        <wlp:install-feature ref="testServer" name="mongodb-2.0" whenFileExists="ignore" acceptLicense="true" />
-
+        <wlp:install-feature ref="testServer" whenFileExists="ignore" acceptLicense="true">
```

Comment:
Hi, can you confirm that the old way of doing this with name="mongodb-2.0" still works so we haven't broken existing scripts.